### PR TITLE
Fix colour of clock icon in ViewProfile

### LINKED ISSUE #3329: View profile: clock icon for last seen is not clearly visible in dark mode
Bildschirmfoto 2023-07-22 um 20 13 13

I think it should become white or inverted in darkmode. maybe there is a way to just give it the text color (the css variable which gets set by the theme).

### PR DESCRIPTION
**Pull request by farooqkz**
Fixes #3329

### CODE REVIEW

File: scss/dialogs/_view_profile.scss [hunk 1]
```diff
@@ -58,5 +58,17 @@ div.profile-info-container {
     &.clickable:hover {
       cursor: pointer;
     }
+
+    & > i.material-icon {
```
Comment (Simon-Laux) [inline scss/dialogs/_view_profile.scss:62]: we might have other material icons, so maybe better add the bulk of this calls to some class maybe named `.material-svg-icon` and then also add a class `.material-icon-schedule`? and do this in the scss file for icons. then we can also reuse the class elsewhere and would just overwrite things like size in viewProfile if even necessary, Basing it on `em` is a good choice so probably such overwrites are not necessary.

Comment (farooqkz) [reply scss/dialogs/_view_profile.scss:62]: Agreed.

Comment (farooqkz) [reply scss/dialogs/_view_profile.scss]: @Simon-Laux How is it now?

File: scss/dialogs/_view_profile.scss [hunk 2]
```diff
@@ -58,5 +58,14 @@ div.profile-info-container {
...
       cursor: pointer;
     }
+
+    & > i.material-icon-schedule {
+      -webkit-mask-image: url(../images/icons/schedule.svg);
+      height: 1.15em;
+      width: 1.15em;
+      -webkit-mask-size: 1.15em;
+      margin-inline-start: 0.3em;
+      margin-inline-end: 0.3em;
```
Comment (Simon-Laux) [inline scss/dialogs/_view_profile.scss:68]: I think you can also put this in the base class as it adapts to the current font size

**Review by Simon-Laux — CHANGES_REQUESTED** [review]
Code looks good and works, but the icon is not entered anymore which does not look good:

before:
Bildschirmfoto 2023-07-26 um 20 59 55
after:
Bildschirmfoto 2023-07-26 um 21 00 05

File: scss/_icons.scss [hunk 3]
```diff
@@ -45,3 +45,19 @@ img.material-icon {
...
+  height: 1.15em;
+  width: 1.15em;
+  -webkit-mask-size: 1.15em;
+  margin-inline-start: 0.3em;
+  margin-inline-end: 0.3em;
+  vertical-align: middle;
+}
+
+i.material-icon-schedule {
+  -webkit-mask-image: url(../images/icons/schedule.svg);
```
Comment (Simon-Laux) [inline scss/_icons.scss:62]:
```suggestion
  -webkit-mask-image: url(../images/icons/schedule.svg);
  transform: translate(0px, -0.07em);
```

adding this makes it look more centred, I have the values from eh other icons above.

Comment (farooqkz) [reply scss/_icons.scss:62]: image
image

It is fine as it is now

The first image is from this PR and the latter from master(with manually whitening the clock icon)

Comment (Simon-Laux) [reply scss/_icons.scss:62]: I think the offset looks better, I tried it because to me it looked a bit off.

**Review by Simon-Laux — APPROVED** [review]

**Pull request merged**